CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 ### Added
 - Public: Added support for Spanish language. The SDK can now be used in Spanish by adding `{language: 'es'}` to the initialization options. If `language` is not present or the wrong locale tag is provided, the language locale will default to `en`.
 - Public: Added support for Spanish language on the SMS body.
+- Internal: Moved `/locales` folder within `src`
```

Comment:
not sure if we need to put it on the changelog. was there a locales folder on the previous version? did it break on IE then?

Reply:
There wasn't a `/locales` folder in previous versions. I agree, no need for changelog.